realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -399,26 +398,26 @@ protected final void copy(ColumnInfo rawSrc, ColumnInfo rawDst) {
     }
 
     public static RealmObjectSchema createRealmObjectSchema(RealmSchema realmSchema) {
+        if (realmSchema.contains("AllTypes")) {
```

Comment:
Any reason to add this? `createRealmObjectSchema` is only called if the Realm file is unversioned (=just created). So this will always be false? https://github.com/realm/realm-java/blob/master/realm/realm-library/src/main/java/io/realm/Realm.java#L461